### PR TITLE
#1443 docs move readme documents

### LINKED ISSUE #1443: docs: move readme documents from secondary entry points to docs
When #1420 has been merged, find every transcluded `README.md` document in a **secondary** entry point, for example `libs/cdk/coercing/README.md` in secondary entry point `@rx-angular/cdk/coercing`.

Move the contents to the document transcluding it, in this case `apps/docs/docs/cdk/coercing/coercing.mdx`.

If there is a partial `_docs.md` document adjacent to it, move its contents into the main document as well, in this case, `coercing.mdx`.

Delete relevant entries from `implicitDependencies` in `nx.json`.

### PR DESCRIPTION
**Pull request by mikelgo**
closes #1443 

Question:
- should i move also the content of CONTRIBUTING.md to `apps/docs/docs/contributing.mdx`? I guess not right?

### CODE REVIEW

**Comment by LayZeeDK** [comment]
> should i move also the content of CONTRIBUTING.md to apps/docs/docs/contributing.mdx? I guess not right?

No, CONTRIBUTING.md gets special treatment by GitHub.

**Comment by LayZeeDK** [comment]
@mikelgo Did you check for links to README.md and docs.md files?

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
libs/eslint-plugin/README.md
and
libs/template/README.md
must be reverted. These are primary entry points that are packaged with the libs to appear on the npm pages.

**Comment by mikelgo** [comment]
> @mikelgo Did you check for links to README.md and docs.md files?

I did for sure not check all the links - but the one's I checked are fine.

**Comment by mikelgo** [comment]
> libs/eslint-plugin/README.md and libs/template/README.md must be reverted. These are primary entry points that are packaged with the libs to appear on the npm pages.

done

**Comment by mikelgo** [comment]
> > @mikelgo Did you check for links to README.md and docs.md files?
> 
> I did for sure not check all the links - but the one's I checked are fine.

I checked with https://github.com/tcort/markdown-link-check.
It found some broken links, but not related to my changes.
I would open a separate issue and fix them in a separate PR

File: libs/eslint-plugin/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # @rx-angular/eslint-plugin
 
-> A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
+A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
```
Comment (LayZeeDK) [inline libs/eslint-plugin/README.md:3]: question: Any reason for this change?

suggestion: Revert the change to leave this file untouched in this PR.
```suggestion
> A set of ESLint rules for building reactive, performant and Zone-less Angular applications.
```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
Small comment. Otherwise, this looks great!

File: libs/eslint-plugin/README.md (hunk 1 above)
Comment (mikelgo) [reply libs/eslint-plugin/README.md]: Oops - no I guess this happened by accident. Good catch!

**Comment by mikelgo** [comment]
> > > @mikelgo Did you check for links to README.md and docs.md files?
> > 
> > 
> > I did for sure not check all the links - but the one's I checked are fine.
> 
> I checked with https://github.com/tcort/markdown-link-check. It found some broken links, but not related to my changes. I would open a separate issue and fix them in a separate PR

Follow up issue created #1491

**Review by edbzn — APPROVED** [review]

**Review by LayZeeDK — APPROVED** [review]
